packages/typescript-react/index.js

```diff
@@ -0,0 +1,37 @@
+const { settings: baseSettings, rules: baseRules } = require('eslint-config-brainbits-typescript');
+
+module.exports = {
+    extends: [
+        'eslint-config-brainbits-react',
+        'eslint-config-brainbits-typescript',
+    ].map(require.resolve),
+    settings: {
+        // Apply special parsing for TypeScript files
+        'import/parsers': {
+            '@typescript-eslint/parser': ['.tsx', ...baseSettings['import/parsers']['@typescript-eslint/parser']],
+        },
+        // Append 'ts' and 'tsx' extensions to Airbnb 'import/resolver' setting
+        'import/resolver': {
+            node: {
+                extensions: ['.tsx', '.jsx', ...baseSettings['import/resolver'].node.extensions],
+            },
+        },
+        // Append 'ts' and 'tsx' extensions to Airbnb 'import/extensions' setting
+        'import/extensions': [...baseSettings['import/extensions'], '.tsx'],
+    },
+    rules: {
```

Comment:
Do we enforce prop types via typescript type definitions?

Reply:
Is there a specific rule for that? I think it should be handled via the no implicit any typescript rule automatically?

Reply:
I don't know. Thats why I'm asking 😅